### ISSUE TITLE
General question and js2 tools compatibility

### ISSUE THREAD

**Issue by marcinant** [thomblake/js3-mode#87]
Hi!

I installed js3 as new and improved version of js2-mode. Then I realized that there are plenty of tools not compatible with js3.

For example: auto-complete - needs hack to make it work with js3, slime-js - depends on js2, js2-refactor, smart-tabs, skewer, ac-js2.... and probably few more I don't know about.

So, my general question is: What is the point of developing and maintaining two similar but not fully compatible javascript modes?

What exactly js3 can do, that js2 cannot?

Is there some precise comparison table?

Is this possible to just extend js2 with improvements taken from js3 and merge these features?

I will switch back to js2 for now as I want to try skewer/slime-js, but it could be nice to have best of these two forks in single one.
